src/server/config.min.ini

```diff
@@ -108,15 +108,11 @@
   server_load_balancer_type = greedy_load_balancer
   min_live_node_count_for_unfreeze = 1
   cold_backup_disabled = false
-  recover_from_replica_server = false
```

Comment:
recovery function test will update it into true dynamically, removing it will make function test failed, I recommend remaining this config.